initfiles/componentfiles/configschema/xsd/esp_service_smc.xsd

```diff
@@ -13,6 +13,19 @@
             <xs:attribute name="warnIfFreeMemoryUnder" type="xs:nonNegativeInteger" use="optional" default="5" hpcc:displayName="Warn Free Memory" hpcc:group="Monitoring" hpcc:tooltip="Available memory under this value is flagged as warning in monitoring output"/>
         </xs:attributeGroup>
 
+        <xs:attributeGroup name="elk">
+            <xs:attribute name="integrateKibana" type="xs:boolean" use="required"  hpcc:defaultInCode="true" hpcc:tooltip="Declare if Kibana dashboard to be integrated in ECLWatch"/>
```

Comment:
This seems like a perfect candidate for an actual default value. Also default should be false

Reply:
This XSD needs to go out for full review. At that time the question of default vs defaultInCode should be addressed. However, remember that a default value will ALWAYS cause it to be written to the XML thus if the code wanted a different default value, each environment XML would need to be edited. 

And, the value is set to true in the original PR.

Reply:
Please go ahead and mark it default